File: TabloidFullStack/TabloidFullStack/client/tabloid/src/components/Comment/Comment.js
import { Link } from "react-router-dom";
import { Table } from "reactstrap";

export const Comment = ({comment}) => {
    
    return(
        <><p className="text-left px-2">Post Title: <strong>{comment.post?.title}</strong></p><Table striped bordered hover>
            <thead>
                <tr>
                    <th>Content</th>
                    <th>Subject</th>
                    <th>Author</th>
                    <th>Creation Date</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td>{comment.content}</td>
                    <td>{comment.subject}</td>
                    <td>{comment.userProfile?.displayName}</td>
                    <td>{comment.createDateTime}</td>
                </tr>
            </tbody>
        </Table></>  
    );
}

File: TabloidFullStack/TabloidFullStack/Models/Comment.cs
using Microsoft.Extensions.Hosting;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace TabloidFullStack.Models
{

    public class Comment
    {
        public int Id { get; set; }

        [Required]
        public string Subject { get; set; }

        [Required]
        public string Content { get; set; }

        public DateTime CreateDateTime { get; set; }

        [Required]
        [DisplayName("Post")]
        public int PostId { get; set; }

        //public Post? Post { get; set; }

        [DisplayName("Author")]
        public int UserProfileId { get; set; }
        public UserProfile? UserProfile { get; set; }

    }
}